src/WebJobs.Script/Description/Node/Script/globalInitialization.js

```diff
@@ -8,6 +8,9 @@ return function (context, callback) {{
         context.handleUncaughtException(err.stack);
     }});
 
+    // TEMP HACK: workaround for https://github.com/tjanczuk/edge/issues/325
```

Comment:
This is really bad. There are differences between process.nextTick and global.setImmediate, you could be breaking third party packages in really, nasty, subtle ways.